pkg/webhook/admission/initializer/interfaces.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Here's an example of a file that was copied from Kubernetes.

I don't know if it's strictly necessary to retain the original copyright line, but Google's guidance suggests so. 
And I know there's tons of other code in cert-manager which is derived from Kubernetes sources, so I'm sorry if it seems nit picky to bring this up here.
Perhaps we should talk about it in the next community meeting?
Perhaps there is guidance from CNCF about whether this is important?

```suggestion
/* 
Copyright 2017 The Kubernetes Authors.

Derived from https://github.com/kubernetes/kubernetes/blob/9d0d2e8ece9bdd0cd8c23be2f36eee5473afc648/staging/src/k8s.io/apiserver/pkg/admission/initializer/interfaces.go
*/
```

> 
> Files that copy lines of code from third-party files, or that are directly translated or derived from third-party files, must retain any copyright notices present in the third-party file(s). A Google copyright notice can be added to the license header as a separate copyright notice, beneath any existing notices, but above the license text.
> 
> Any files that are exact copies of files taken from a third-party source must retain any existing copyright notices and must not include any new copyright notices.

https://opensource.google/docs/copyright/#files-derived-from-third-party-code